controller/ProctorManager.php

```diff
@@ -180,23 +181,10 @@ public function createProctorForm(){
         
         return $this->returnJson(array(
```

Comment:
The `returnJson` method does not return a value, so it is useless to apply a `return` here.